larray/session.py

```diff
@@ -406,7 +409,7 @@ def dump(self, fname, names=None, engine='auto', display=False, **kwargs):
             arrays = [(k, v) for k, v in arrays if k in names_set]
         handler.dump_arrays(arrays, display=display, **kwargs)
 
-    def dump_hdf(self, fname, names=None, *args, **kwargs):
+    def to_hdf(self, fname, names=None, *args, **kwargs):
```

Comment:
needs changelog, a dummy method with the old name with a deprecation warning and a mention of the #217 issue in the commit message + changelog :)